### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vonbearshark**

### CODE REVIEW

File: talk-4-advanced-js-2/README.md [hunk 1]
```diff
@@ -11,8 +11,13 @@
 ### Callbacks
 - 
 
+### TCO
+- As it stands, JavaScript is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in `Maximum call stack size exceeded`
```
Comment (alexlafroscia) [inline talk-4-advanced-js-2/README.md:15]: > is not tail-call optimized for recursive calls. ES6 stands to change that, but it's still something to keep in mind with recusive functions in JavaScript, which, as it stands, is not tail-call optimized for recursive calls.

I'm not sure if this is supposed to be notes or the basis for slides, but that's a little repetitive.

Comment (alexlafroscia) [inline talk-4-advanced-js-2/README.md:15]: Actually now that I'm reading it again, is it all a play on the fact that you're talking about recursion? That went right over my head :sob:

Comment (vonbearshark) [inline talk-4-advanced-js-2/README.md:15]: Some would say it's even recursive haha

Comment (vonbearshark) [inline talk-4-advanced-js-2/README.md:15]: Haha it's late on the East coast. I understand!

**Pull request merged**